src/stats/hist.jl

```diff
@@ -72,6 +72,9 @@ function Makie.plot!(plot::Hist)
     edges = lift(values, plot.bins) do vals, bins
         if bins isa Int
             mi, ma = float.(extrema(vals))
+            if mi == ma
+                return nothing
```

Comment:
Can't we just set

```julia
mi, ma = mi - 0.5, ma + 0m5
```

and keep all the remaining logic the same? We still get the arbitrary bar of width `1` centered around the unique value, but avoid having to keep two different code paths.

In particular, I think having 2 separate code paths can cause issues with normalization options (for example with `normalization = :pdf` weights should be `1` instead of `values[][1]`).

Reply:
Good idea, thanks :)